synchronicity/synchronizer.py

```diff
@@ -701,6 +718,17 @@ def nowrap(self, obj):
         setattr(obj, self._nowrap_attr, True)
         return obj
 
+    def no_input_translation(self, obj):
+        setattr(obj, self._no_input_translation_attr, True)
```

Comment:
FWIW I found that the multiple negation makes this harder to reason about. I.e. `self._no_input_translation_attr = False` means "do input translation", which is the default case. It would be annoying to go through and invert everything though, so up to you.

Reply:
Good point